pkg/commands/models/branch.go

```diff
@@ -10,6 +10,7 @@ type Branch struct {
 	Pushables   string
 	Pullables   string
 	Head        bool
+	Gone        bool
```

Comment:
The name `Gone` is a little misleading: it suggests this branch itself is gone which is not the case. What are your thoughts on `UpstreamGone`. I'd also move this up one line so it's next to `Pullables` and `Pushables` given how related the are.

Reply:
Yeah, this is a good one. I simply use the name because it is what appears on Git, but totally agree `UpstreamGone` would be much clearer. This has also been moved up.